### PR TITLE
Add spinner to UCR report builder form save button

### PR DESCRIPTION
**Pull request by NoahCarnahan**
as per UAT feedback.
@nickpell 

screen shot 2015-06-11 at 3 22 23 pm

Ticket: http://manage.dimagi.com/default.asp?171142

### CODE REVIEW

File: corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html [hunk 1]
```diff
@@ -115,5 +115,17 @@
         };
         var formViewModel = new configForm();
         ko.applyBindings(formViewModel, $("#report-config-form").get(0));
+
+        // Give feedback to user while form is submitting.
+        // Clicking the save button isn't an ajax request, but it can be slow to save.
+        $("#report-config-form [name='submit']").click(function() {
+            $(this).hide();
+            $(this).after(
```
Comment (nickpell) [inline corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html]: any reason this doesn't use ``disable-on-submit```?

File: corehq/apps/userreports/reports/builder/forms.py [hunk 2]
```diff
@@ -346,7 +346,9 @@ def __init__(self, report_name, app_id, source_type, report_source_id, existing_
         self.helper.attrs['data_bind'] = "submit: submitHandler"
         self.helper.form_id = "report-config-form"
 
-        buttons = [crispy.Submit('submit', self.button_text)]
+        buttons = [
+            crispy.HTML("<button type='submit' class='btn btn-primary disable-on-submit'>{}</a>".format(self.button_text))
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E501 line too long (122 > 115 characters)

File: corehq/apps/userreports/reports/builder/forms.py [hunk 3]
```diff
@@ -346,7 +346,13 @@ def __init__(self, report_name, app_id, source_type, report_source_id, existing_
         self.helper.attrs['data_bind'] = "submit: submitHandler"
         self.helper.form_id = "report-config-form"
 
-        buttons = [crispy.Submit('submit', self.button_text)]
+        buttons = [
+            crispy.HTML(
+                "<button type='submit' "
+                    "class='btn btn-primary disable-on-submit'"
```
Comment (dimagimon) [inline corehq/apps/userreports/reports/builder/forms.py]: E131 continuation line unaligned for hanging indent

File: corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html (hunk 1 above)
Comment (NoahCarnahan) [inline corehq/apps/userreports/templates/userreports/partials/report_builder_configure_report.html]: Awesome idea. updated.

**Pull request merged**